src/Storefront/Resources/app/storefront/test/plugin/address-manager/address-manager.plugin.test.js

```diff
@@ -0,0 +1,250 @@
+import AddressManagerPlugin from 'src/plugin/address-manager/address-manager.plugin';
+
+/**
+ * @package checkout
+ */
+
+let addressManager;
+
+describe('AddressManagerPlugin test', () => {
+    afterEach(() => {
+        document.body.innerHTML = '';
+        jest.useRealTimers();
+    });
+
+    test('plugin initializes', () => {
+        create();
```

Comment:
Do not use a test case persistent variable for a value that shouldn't persist that long
```suggestion
describe('AddressManagerPlugin test', () => {
    let addressManager;

    afterEach(() => {
        document.body.innerHTML = '';
        jest.useRealTimers();
    });

    test('plugin initializes', () => {
        addressManager = create();
```